hexane/force_matching/run.sh

```diff
@@ -0,0 +1,37 @@
+#! /bin/bash -e
```

Comment:
ok. I actually now put a separate force_matching folder for spce and methanol as well